Dockerfile.processing_image

```diff
@@ -11,17 +11,17 @@ ENV PIP_NO_CACHE_DIR=1
 
 # HACK: Create a no-op npm; npm build not needed for backend-only operations
 # The no-op npm touches some files that are the cellxgene install script expects to exist
-RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
- && echo '#!/bin/sh\nmkdir -p build/static && touch build/static/.gitignore build/index.html build/csp-hashes.json' > /usr/bin/npm && chmod 755 /usr/bin/npm \
- && cd single-cell-explorer \
- && make build-for-server-dev \
- && cd .. \
- && rm -rf single-cell-explorer client build/client /usr/bin/npm
+# RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
+#  && echo '#!/bin/sh\nmkdir -p build/static && touch build/static/.gitignore build/index.html build/csp-hashes.json' > /usr/bin/npm && chmod 755 /usr/bin/npm \
+#  && cd single-cell-explorer \
+#  && make build-for-server-dev \
+#  && cd .. \
+#  && rm -rf single-cell-explorer client build/client /usr/bin/npm
```

Comment:
prove to me we need this...

Reply:
99.99% sure we don't need it. We'll find out for sure later today. 😄

Reply:
@metakuni Anything break?